cli/cmd/compliance_azure.go

```diff
@@ -202,18 +202,29 @@ To run an ad-hoc compliance assessment use the command:
 				}
 			}
 
-			cli.StartProgress(" Getting compliance report...")
-			response, err := cli.LwApi.Compliance.GetAzureReport(config)
-			cli.StopProgress()
-			if err != nil {
-				return errors.Wrap(err, "unable to get azure compliance report")
-			}
+			var (
+				report   api.ComplianceAzureReport
+				cacheKey = fmt.Sprintf("compliance/aws/%s/%s/%s",
```

Comment:
@dmurray-lacework crap, this is wrong 😢 , this should be `compliance/gcp/` or `compliance/google`